spec/features/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+# [] Visiting the `/landmarks/1` path should show the landmark details for a landmark with the ID of 1.
+
+feature "user deletes a review" do
+  scenario "user views landmark and deletes review" do
+    user2 = FactoryGirl.create(:user)
+    user3 = FactoryGirl.create(:user)
+    user4 = FactoryGirl.create(:user)
+
+    statue_of_liberty = Landmark.create(
+      name: "Statue of Liberty",
+      location: "New York",
+      image: "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Statue_of_Liberty_7.jpg/500px-Statue_of_Liberty_7.jpg",
+      description: "A colossal neoclassical sculpture on Liberty Island"
+    )
+
+    Review.create(
+      body: "SO FRESH AND SO GREEN GUYS",
+      landmark: statue_of_liberty,
+      user: user2,
+      votes: "3"
+    )
+
+    Review.create(
+      body: "I don't even think the statue can read.",
+      landmark: statue_of_liberty,
+      user: user3,
+      votes: "56"
+    )
+
+    Review.create(
+      body: "I got tired walking up the stairs.",
+      landmark: statue_of_liberty,
+      user: user4,
+      votes: "-3"
+    )
```

Comment:
This whole block (while hilarious) could be made much more succinct with a sequence factory or using factories and `let` statements with particular traits.